src/App.tsx

```diff
@@ -1,14 +1,45 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { People } from './components/People';
+
+const initialPeople: Person[] = peopleFromServer.map(person => ({
```

Comment:
The `.map` operation here is redundant, as you're spreading each person object into a new one without any modifications. Consider using the original `peopleFromServer` array instead.